examples/demos/components/Editors/DropDownEditor.tsx

```diff
@@ -8,26 +8,21 @@ interface Option {
   text: string;
 }
 
-interface DropDownEditorProps<TRow> {
-  column: Column<TRow>;
+interface DropDownEditorProps<TRow> extends EditorProps<string, TRow> {
   options: Array<Option | string>;
-  value?: string;
-  onCommit(): void;
 }
 
-type DropDownEditorHandle = Pick<Editor<{[key: string]: string | undefined}>, 'getInputNode' | 'getValue'>;
+type DropDownEditorHandle = Editor<string>;
 
-function DropDownEditorWithRef<TRow>({ column, value, onCommit, options }: DropDownEditorProps<TRow>, ref: React.Ref<DropDownEditorHandle>): JSX.Element {
+function DropDownEditor<TRow>({ value, onCommit, options }: DropDownEditorProps<TRow>, ref: React.Ref<DropDownEditorHandle>) {
   const selectRef = useRef<HTMLSelectElement>(null);
 
   useImperativeHandle(ref, () => ({
     getInputNode() {
       return selectRef.current;
     },
     getValue() {
-      return {
-        [column.key]: selectRef.current?.value
-      };
+      return selectRef.current!.value;
```

Comment:
Would this not fail?
https://github.com/adazzle/react-data-grid/blob/131a9d3f13c39527c19adb8210823bd5d057b53c/examples/demos/example13-all-features.tsx#L186

Reply:
Mmh, might be a type issue somewhere, as it wasn't allowing me to return an object.

Reply:
Nevermind, I've fixed it now.